Makefile

```diff
@@ -28,12 +29,12 @@ GRPID   := $(shell id -g)
 .PHONY: install-hooks
 install-hooks:
 	@echo "===> Copying Antrea git hooks to local <==="
-	install hack/git_client_side_hooks/pre-commit .git/hooks/
+	install $(GIT_HOOKS)
 
 .PHONY: uninstall-hooks
 uninstall-hooks:
 	@echo "===> Removing Antrea git hooks from local <==="
-	rm .git/hooks/pre-commit
+	rm $(GIT_HOOKS)	
```

Comment:
This is also wrong as it is. It will try to remove `hack/git_client_side_hooks/pre-commit` and `hack/git_client_side_hooks/pre-push`, which is not what we want. We want to remove `.git/hooks/pre-commit` and `.git/hooks/pre-push`.